src/renderer/shared/ui/LabelHelpbox/LabelHelpBox.tsx

```diff
@@ -1,22 +1,29 @@
 import { PropsWithChildren } from 'react';
 
-import { BodyText, Icon } from '@renderer/shared/ui';
+import { FootnoteText, Icon } from '@renderer/shared/ui';
 import { cnTw } from '@renderer/shared/lib/utils';
 
 type Props = {
   className?: string;
+  disabled?: boolean;
 };
 
-export const LabelHelpBox = ({ className, children }: PropsWithChildren<Props>) => (
-  <div
+export const LabelHelpBox = ({ className, children, disabled = false, ...props }: PropsWithChildren<Props>) => (
+  <button
     className={cnTw(
-      'flex gap-x-1 items-center rounded-md py-0.5 px-2 group outline-offset-1',
-      'bg-secondary-button-background hover:bg-secondary-button-background-hover active:bg-secondary-button-background-active',
+      'flex gap-x-1 items-center rounded-md py-1.5b px-2b group outline-offset-1',
+      'bg-button-secondary-default hover:bg-button-secondary-hover active:bg-button-secondary-active',
       className,
     )}
+    disabled={disabled}
     data-testid="labelHelpbox"
+    {...props}
   >
-    <BodyText>{children}</BodyText>
-    <Icon name="learn-more" className="group-hover:text-icon-hover group-active:text-icon-active" size={16} />
-  </div>
+    <FootnoteText className={cnTw('text-text-primary', disabled && 'text-text-tertiary')}>{children}</FootnoteText>
```

Comment:
This is optional, could leave as you've done
```suggestion
    <FootnoteText className={disabled ? 'text-text-tertiary' : 'text-text-primary'}>{children}</FootnoteText>
```